speech/recognize.js

```diff
@@ -162,6 +165,16 @@ function asyncRecognize (filename, encoding, sampleRateHertz, languageCode) {
     .then((results) => {
       const transcription = results[0].results[0].alternatives[0].transcript;
       console.log(`Transcription: ${transcription}`);
+      results[0].results[0].alternatives[0].words.forEach((wordInfo) => {
+        // NOTE: If you have a time offset exceeding 2^32 seconds, use the
+        // wordInfo.{x}Time.seconds.high to calculate seconds.
+        let startSecs = `${wordInfo.startTime.seconds.low}` + `.` +
+            (wordInfo.startTime.nanos / 100000000);
+        let endSecs = `${wordInfo.endTime.seconds.low}` + `.` +
+            (wordInfo.endTime.nanos / 100000000);
+        console.log(`Word: ${wordInfo.word}`);
+        console.log(`\t ${startSecs} secs - ${endSecs} secs`);
```

Comment:
You'll need to update the test for this method, similar to what was done [here](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/speech/system-test/recognize.test.js#L64)